<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="1" rowspan="23">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="maximumSize">
       <size>
        <width>3000</width>
        <height>3000</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
     </widget>
    </item>
    <item row="21" column="0" alignment="Qt::AlignLeft">
     <widget class="QCheckBox" name="HideTestButtonCheckBox">
      <property name="text">
       <string>Hide test buttons</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_end_city">
      <property name="placeholderText">
       <string>Please enter an end city</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_add_city">
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QPushButton" name="pushButton_test_draw_street">
      <property name="text">
       <string>Test Draw Streets</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_input">
      <property name="text">
       <string>Your Input</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton_test_abstract_map">
      <property name="text">
       <string>Test Abstract Map</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QPushButton" name="pushButton_test_draw_button">
      <property name="text">
       <string>Test Draw City</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pushButton_Fill_Map">
      <property name="text">
       <string>Fill Map</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pushButton_add_street">
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLineEdit" name="lineEdit_test_input">
      <property name="placeholderText">
       <string>Test line editor</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="pushButton_test_input">
      <property name="toolTip">
       <string>Test Input</string>
      </property>
      <property name="text">
       <string>Test Input</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Test Draw Cities</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QPushButton" name="pushButton_test_add_street">
      <property name="text">
       <string>Test Add Streets</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Test Add City</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton_test_simple_dijkstra">
      <property name="text">
       <string>Test simple Dijkstra</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_start_city">
      <property name="placeholderText">
       <string>Please enter a start city</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_search_shortest_path">
      <property name="toolTip">
       <string>Search shortest path with Dijkstra</string>
      </property>
      <property name="text">
       <string>Search shortest path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionTrenner"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionClear_Scene"/>
    <addaction name="actionScene_Test"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTrenner">
   <property name="text">
    <string>Trenner hinzufuegen</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScene_Test">
   <property name="text">
    <string>Scene Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
